x-pack/test/api_integration/apis/secops/overview_network.ts

```diff
@@ -0,0 +1,125 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import expect from '@kbn/expect';
+import { overviewNetworkQuery } from '../../../../plugins/secops/public/containers/overview/overview_network/index.gql_query';
+import { GetSiemOverviewNetworkQuery } from '../../../../plugins/secops/public/graphql/types';
+import { KbnTestProvider } from './types';
+
+const overviewNetworkTests: KbnTestProvider = ({ getService }) => {
+  const esArchiver = getService('esArchiver');
+  const client = getService('secOpsGraphQLClient');
+  describe('Overview Network', () => {
+    describe('With filebeat', () => {
+      before(() => esArchiver.load('filebeat/default'));
+      after(() => esArchiver.unload('filebeat/default'));
+
+      const FROM = new Date('2000-01-01T00:00:00.000Z').valueOf();
+      const TO = new Date('3000-01-01T00:00:00.000Z').valueOf();
+
+      const expectedResult = {
+        packetbeatFlow: 0,
+        packetbeatDNS: 0,
+        filebeatSuricata: 4547,
+        filebeatZeek: 0,
+        auditbeatSocket: 0,
+        __typename: 'SiemOverviewNetworkData',
+      };
+
+      it('Make sure that we get SiemOverviewNetwork data', () => {
+        return client
+          .query<GetSiemOverviewNetworkQuery.Query>({
+            query: overviewNetworkQuery,
+            variables: {
+              sourceId: 'default',
+              timerange: {
+                interval: '12h',
+                to: TO,
+                from: FROM,
+              },
+            },
+          })
+          .then(resp => {
+            const overviewNetwork = resp.data.source.SiemOverviewNetwork;
+            expect(overviewNetwork).to.eql(expectedResult);
+          });
+      });
+    });
+
+    describe('With packetbeat', () => {
```

Comment:
Question: the test makes sense here, but I will expect that if we are testing packetBeat, I should see some packetbeatFlow or packetbeatDns. You should try to use to load another Archiver here for packetBeat